core/src/main/java/org/elasticsearch/search/suggest/DirectSpellcheckerSettings.java

```diff
@@ -130,4 +130,20 @@ public void minDocFreq(float minDocFreq) {
         this.minDocFreq = minDocFreq;
     }
 
+    @Override
+    public String toString() {
+        return "[" +
```

Comment:
nit: currently toString in ToXContentToBytes is called, which prints the json representation. Unless there is something missing from that output, I'd say we keep that, so for example when this is printed in some error message users see the familiar json from the query DSL.